### ISSUE TITLE
Uploading new CSV file to old database causes crash

### ISSUE THREAD

**Issue by MerielP** [ox-it/ords#766]
To reproduce this bug, find an old database (I am guessing that in this context, 'old' means created before the ODBC work) that has at least one CSV file uploaded to it, on either Dev or Staging. Try to upload another CSV file. The result is a distinctly unfriendly error message.

![csv file upload error message](https://cloud.githubusercontent.com/assets/13660306/13176439/3500ea3a-d70a-11e5-8582-d470c80a6867.PNG)

This doesn't happen with newly created databases, and doesn't seem to be an issue at all on App. My guess is therefore that this is another instance of something having been broken by the ODBC work.

(Marked critical on James's advice - needs fixing before the new version is deployed to App.)

**Comment by thestoat** [comment]
Please take another look at this issue. The crash should now be fixed but I think the crash was simply masking an underlying problem. The implication I draw from this issue is that the upload should have worked - the crash tells me it didn't. When you perform this test again, please let me know the exact date and time when the error happens so I can check logs.

<Added:>
I've just noticed this problem was reported against staging. Small fixes like the one I added won't easily gfet to Staging without a lot of work from Sysdev that they are understandably reluctant to do. If you cannot reproduce this problem on Dev then please retry (on Staging) and let me know
1. The date and the time when you performed the re-test 
2. The full URL (in your browser) immediately before you uploaded the database

**Comment by MerielP** [comment]
I've just tried this again on Dev (in the database 'Testing CSV upload and new table creation' in 'Sample project'). It still didn't work, but this time I got this error message:

![csv file upload error message 2](https://cloud.githubusercontent.com/assets/13660306/13216099/b8c070c2-d951-11e5-9c7c-574d65b5f5df.PNG)

I tried this twice, and the same thing happened both times. The clock on my computer said 10.44.
